package.json

```
@@ -36,16 +36,26 @@
       "name": "Michael Prentice",
       "email": "michael@xlts.dev",
       "url": "https://github.com/splaktar"
+    },
+    {
+      "name": "Jaroslaw Zolnowski",
+      "email": "jarek@xlts.dev",
+      "url": "https://github.com/jzolnowski"
     }
   ],
-  "main": "index.js",
+  "main": "main.js",
   "scripts": {
-    "start": "webpack-dev-server --open --mode development",
+    "pre-commit": "lint-staged && npm run lint",
+    "build:templates": "gulp templates",
+    "start:server": "webpack-dev-server --open --mode development --config config/webpack.dev.mjs",
+    "gulp:watch": "gulp watch",
+    "start": "npm run build:templates && npm-run-all -p start:server gulp:watch",
```

Nit: This can be shortened as:

```suggestion
    "start": "npm run build:templates && run-p start:server gulp:watch",
```